md_main.py

```diff
@@ -0,0 +1,126 @@
+import argparse
```

Comment:
it would be nice to use [`click`](https://click.palletsprojects.com/en/8.1.x/), which we use in many other projects. very easy to get command line apps set up